spec2sdk/models/entities.py

```diff
@@ -274,11 +342,8 @@ def imports(self) -> Sequence[Import]:
         return ()
 
     def render(self) -> str:
-        if self.name:
-            root_type = f"list[{self.inner_py_type.type_hint}]"
-            return f"type {self.name} = {root_type}"
-        else:
-            return ""
+        root_type = f"list[{self.inner_py_type.type_hint}]"
```

Comment:
For arrays there are `minItems` and `maxItems` constraints. Would it be hard to add them here?

Reply:
I've added `minItems` and `maxItems` constraints for array type. It was not very difficult. Adding `uniqueItems` is a bit harder because it requires to create custom validator. It's not a big deal, but I found that current implementation doesn't allow to render extra code for a type. This can be improved later.